debian/qubes-core-agent.install

```diff
@@ -8,6 +8,7 @@ etc/default/grub.d/30-qubes.cfg
 etc/fstab
 etc/needrestart/conf.d/50_qubes.conf
 etc/profile.d/qt_x11_no_mitshm.sh
+etc/qubes-rpc/admin.vm.TerminalDispVM
```

Comment:
See https://github.com/QubesOS/qubes-core-admin/pull/259#discussion_r281393567 comment about the name.